java/client/src/main/java/glide/api/commands/ScriptingAndFunctionsClusterCommands.java

```diff
@@ -266,6 +269,111 @@ CompletableFuture<ClusterValue<Map<String, Object>[]>> functionList(
      */
     CompletableFuture<String> functionDelete(String libName, Route route);
 
+    /**
+     * Returns the serialized payload of all loaded libraries.<br>
+     * The command will be routed to all primary nodes.
+     *
+     * @since Redis 7.0 and above.
+     * @see <a href="https://redis.io/docs/latest/commands/function-dump/">redis.io</a> for details.
+     * @return The serialized payload.
+     * @example
+     *     <pre>{@code
+     * ClusterValue<byte[]> data = client.functionDump().get();
+     * // data stores serialized dump from all primary nodes
+     * // it could be saved to restore loaded functions on any Redis instance
```

Comment:
to be consistent with the other docs:
```suggestion
     * // now data could be saved to restore loaded functions on any Redis instance
```